tools/UnitTestApp/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@fluentui/fluentui-charting-contrib",
+    "version": "1.0",
+    "description": "Experimental React charting Contribution environment for building experiences for React charting library.",
```

Comment:
update name and description of package.json

Reply:
done